x-pack/legacy/plugins/actions/server/builtin_action_types/webhook.ts

```diff
@@ -120,9 +149,25 @@ function successResult(data: any): ActionTypeExecutorResult {
   return { status: 'ok', data };
 }
 
+function errorRequestInvalid(id: string, message: string): ActionTypeExecutorResult {
+  const errMessage = i18n.translate('xpack.actions.builtin.webhook.invalidRequestErrorMessage', {
+    defaultMessage:
+      'an error occurred in webhook action "{id}" calling a remote webhook: {message}',
+    values: {
+      message,
+      id,
+    },
+  });
+  return {
+    status: 'error',
+    message: errMessage,
+  };
+}
+
 function errorResultInvalid(id: string, message: string): ActionTypeExecutorResult {
   const errMessage = i18n.translate('xpack.actions.builtin.webhook.invalidResponseErrorMessage', {
-    defaultMessage: 'an error occurred in action "{id}" calling a remote webhook: {message}',
+    defaultMessage:
+      'an error occurred in webhook action "{id}" calling a remote webhook: {message}',
```

Comment:
This particular default message template seems like it's repeated a few times in the module - I find it's good practice to always make them somewhat unique.  In the case when you're diagnosing a failure, and have a log message that can get generated from multiple code paths, you're not going to know which one it went down to get there.  If the messages are unique, you will.

Reply:
It's funny you say that, because I specifically aligned them (originally they were different) so that they read similarly with just the error detail being different.
If the Id ('xpack.actions.builtin.webhook.invalidResponseErrorMessage') not visible in any way? Is it just for i18n?
I had assumed (clearly incorrectly) that _that_ was the identifier, but if only the message makes it into the log then yeah, I'll make this clearer.

Reply:
This is now updated so each message includes a "subject" which is unique.